qcodes/plots/base.py

```diff
@@ -81,7 +81,11 @@ def add(self, *args, updater=None, **kwargs):
                 into `x`, `y`, and optionally `z`, these are passed along to
                 self.add_to_plot.
                 To use custom labels and units pass for example:
-                    >>> plot.add(x=set, y=amplitude, xlabel=("set", "s"), ylabel= ("Amplitude", "V"))
+                    plot.add(x=set, y=amplitude, 
```

Comment:
Good docstring :+1: 

I guess it would not be possible to make them explicit keyword arguments of the add function that then always get passed on to the relevant subfunction? I think this would improve usability, as the shift-tab functionality would then show this as default keyword arguments. 

That being said, I consider that an enhancement that goes beyond the scope of this PR, so maybe for a next time.